docs/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # Changelog
 
+## v.0.6.0 (2021-05-04)

```

Comment:
Could be put also just "WIP" or "0.6.0 (TBD)", but this is OK for now. Just have to remember to update this when the version will actually be released.